components/job-orchestration/job_orchestration/executor/compress/celery.py

```
@@ -1,9 +1,22 @@
 from celery import Celery
+from celery.signals import worker_process_init, worker_process_shutdown
+from clp_py_utils.telemetry import init_telemetry, shutdown_telemetry
 
 from job_orchestration.executor.compress import celeryconfig
 
 app = Celery("compress")
 app.config_from_object(celeryconfig)
 
+
+@worker_process_init.connect
+def setup_telemetry(**kwargs) -> None:
```

Ruff (0.15.15)

[warning] Missing type annotation for `**kwargs`

(ANN003)